vpn.py

```diff
@@ -22,7 +22,10 @@ def parse_args():
     parser.add_argument('--create-hma-configs', dest='create_HMA',
                         action="store_true",
                         help='Create the openvpn config files for HMA')
-    parser.add_argument('--log-file', dest='log_file', default='vpn-log.log',
+    parser.add_argument('--exclude', "-e", dest='exclude_list', default=None,
```

Comment:
Talking to Citizen Lab folks, it seems like that some of the VPNs are not very representative (endpoints in datacenters, etc.) of the network conditions in those countries. This way we have the option exclude them (though we may not always do it).
I'll add the feature to exclude certain IP addresses too.